api/src/OpenTelemetry/Environment.hs

```diff
@@ -0,0 +1,17 @@
+module OpenTelemetry.Environment (
+  lookupBooleanEnv,
+) where
+
+import qualified Data.Char as C
+import System.Environment (lookupEnv)
+
+
+{- | Does the given value of an environment variable correspond to "true" according
+to [the OpenTelemetry specification](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#boolean-value)?
+-}
+isTrue :: String -> Bool
+isTrue = ("true" ==) . map C.toLower
+
+
+lookupBooleanEnv :: String -> IO (Maybe Bool)
+lookupBooleanEnv = fmap (fmap isTrue) . lookupEnv
```

Comment:
> Any value not explicitly defined here as a true value, **including unset** and empty values, MUST be interpreted as false

I think it would be better to handle all aspects of the spec here, rather than push a `Nothing` to the callers (to potentially be misinterpreted).


```suggestion
lookupBooleanEnv :: String -> IO Bool
lookupBooleanEnv = fmap (maybe False isTrue) . lookupEnv
```

Reply:
From spec: _Any value not explicitly defined here as a true value, including unset and empty values, MUST be interpreted as false._
I see :)

Reply:
done

Reply:
Good catch!